crypto/x509v3/v3_asid.c

```diff
@@ -830,6 +859,10 @@ static int asid_validate_path_internal(X509_STORE_CTX *ctx,
  */
 int X509v3_asid_validate_path(X509_STORE_CTX *ctx)
 {
+    if (ctx->chain == NULL
```

Comment:
> We are allowed to return an error if we detect one!

It's a trick question! If being NULL is not permitted option, then we are better off *not* checking.